features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos1
+    Then the image is listed as active
+    And this image has non-empty ID
+    And its ID can be used to show the image info
+    When I delete the image
```

Comment:
@aspiers I don't agree that removing of an image is different from testing. Everything what is being done should be captured in the scenario text. I consider that as best practice, otherwise we end up with a lot action hidden before the our eyes when reading the feature and spenario specifications.

It's also not practical to put that into an another scenario because every scenario must be independent, in this case we would create dependency by calling one scenario from the other. In case the latter fails, you end up finding the reason why this particular scenario was run. It is clear e.g. for a `Background`, not for other cases.

I would not put constraints on what is actual testing and what not. If the scenario fails while the image is being deleted, it's straight forward to see it, more effort is actually not needed compared to a separate scenario having failed, the reason will be the same.

Additionally, I would like to avoid using hooks as it could end up messy,  and rather keep the scenarios' as simple as possible and build mainly on the expressivnes of Gherkin language.